react-app/src/library/services/vendor-implementations/vendor-implementation.ts

```diff
@@ -39,6 +40,10 @@ export abstract class VendorImplementation extends (EventEmitter as { new(): Str
     throw new Error(`${this.vendorName} - isDeviceAttatched getter not implemented`);
   }
 
+  isSupported (): boolean {
+    return true;
```

Comment:
If this is always returning true, is there really a point to having it?  Or is it to ensure any future additions are of type Vendor Implementation?